Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -425,11 +425,13 @@ public function __construct()
             'Change the permissions of either "<strong>app/logs/</strong>" or  "<strong>var/logs/</strong>" directory so that the web server can write into it.'
         );
 
-        $this->addPhpIniRequirement(
-            'date.timezone', true, false,
-            'date.timezone setting must be set',
-            'Set the "<strong>date.timezone</strong>" setting in php.ini<a href="#phpini">*</a> (like Europe/Paris).'
-        );
+        if (version_compare($installedPhpVersion, '7.0.0', '<')) {
```

Comment:
Well, at least not in this file. Here, checks are always performed against that `$installedPhpVersion` variable, for instance:

https://github.com/sensiolabs/SensioDistributionBundle/blob/c4c5caff48a2f3b17e3cc988e2b9d7a785779f39/Resources/skeleton/app/SymfonyRequirements.php#L486

In fact, `PHP_VERSION_ID` doesn't even appear inside this file.